Core1Platform/HelpAndSupport/02_CreateTicket.md

```diff
@@ -0,0 +1,51 @@
+# Create ticket
+
+The creation of a ticket is necessary, if something happens in your *Actindo Core1 Platform* that you think may be a bug in the Actindo software. In this case, the Actindo team has the chance to solve the bug and can provide an error correction to you and all other customers.   
+For example, one of the following situations may be a reason to create a ticket:   
+- A workflow is failing
+- The main account is not reachable
+- Products show incorrect information on the frontend   
+
+> [INFO] Before creating a ticket, it may be useful to first check whether another users have not already reported the discovered issue. To do this, you can check the *Consulting* (MPS agreement (Managed Professional Services)) or the *Support* (older agreement) module.
```

Comment:
```suggestion
> [Info] Before creating a ticket, it may be useful to first check whether other users have already reported the discovered issue. To do this, you can check the *Consulting* (MPS  (Managed Professional Services) agreement) or the *Support* (older agreement) module.
```


another users --> other users
https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/other-others-the-other-or-another?q=another

INFO -> Info (durchgängig)

Reply:
d